games/supertuxkart/Portfile

```diff
@@ -19,9 +19,10 @@ use_xz              yes
 
 homepage            http://supertuxkart.sourceforge.net/
 
-checksums           md5     8de5455b8fdbb92679e302b76c9041cf \
-                    rmd160  4d32b61c99d6043c8007badd21bb79d38216cc4b \
-                    sha256  d8014e7106ba84f98b5ec5f146249dcffc284fc4083f8f237ff420b9e2219cb0
+checksums           md5     b11b2e477bffb62de3c4c533fa22cf3c \
```

Comment:
It's probably no longer necessary/recommended to specify md5 or sha1 signatures, especially if rmd160, sha256, and size are present.

Reply:
right; was the copy and paste of `port -d checksum` and this is the first time that it produces md5. Probably because it notices that before was md5?
anyway, removed. thank you